### ISSUE TITLE
'Auto' setting for max streaming bitrate not working as expected

### ISSUE THREAD

**Issue by SteveDinn** [jellyfin/jellyfin-androidtv#2601]
### Describe the bug

I had the `max streaming bitrate` setting in its default value of `auto` when I tried to play a 4K movie on my CCwGTV. When it played, I found it was being transcoded when it was capable of direct playing. In trying to figure out why this might have been, I switched the `auto` setting to the maximum numerical value of `120 Mbit/s`, and it didn't try to transcode anymore, and successfully direct played.

My CCwGTV is connected via Gigabit Ethernet so none of my files should be getting transcoded because of a bandwidth limitation.

Since switching the `max streaming bitrate` to `120 Mbit/s` resulted in the file direct playing, I have to assume something is wonky with the `Auto` setting, or perhaps the detection of available bandwidth.

Key log information: `PlayMethod=Transcode, TranscodeReason=ContainerBitrateExceedsLimit`

### Logs

With `Max streaming bitrate` set to `Auto`:
```
[2023-03-18 01:36:24.685 -03:00] [INF] [45] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="AndroidTV-libVLC", Path="/media/Movies/Sci Fi & Fantasy/Lord of the Rings/The Return of the King/The Return of the King - Extended Edition 2160p.mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=Transcode, TranscodeReason=ContainerBitrateExceedsLimit ) "media:/videos/b2c50f9b-7844-a7f2-7820-aef7f9bc5f07/stream.mkv?MediaSourceId=b2c50f9b7844a7f27820aef7f9bc5f07&VideoCodec=h264&AudioCodec=aac,mp3&AudioStreamIndex=1&VideoBitrate=6522423&AudioBitrate=1536000&MaxFramerate=23.976025&api_key=<token>&SubtitleMethod=Encode&CopyTimestamps=true&RequireAvc=false&Tag=d7cfb819e098d34fe79d27da72b86a5b&hevc-level=153&hevc-profile=main10&h264-profile=high,main,baseline,constrainedbaseline&h264-level=51&aac-audiochannels=8&TranscodeReasons=ContainerBitrateExceedsLimit"
```
With `Max streaming bitrate` set to `120 Mbit/s`:
```
[2023-03-18 01:55:49.064 -03:00] [INF] [25] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="AndroidTV-ExoPlayer", Path="/media/Movies/Sci Fi & Fantasy/Lord of the Rings/The Return of the King/The Return of the King - Extended Edition 1080p.mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/b2c50f9b-7844-a7f2-7820-aef7f9bc5f07/stream.mkv?MediaSourceId=b4426bf54e410eccb435797beaf25aec&Static=true&VideoCodec=hevc&AudioCodec=eac3&AudioStreamIndex=0&api_key=<token>&SubtitleMethod=Encode&Tag=764e66a5e37317a92af28f55138c00a4"
```

### Application version

0.15.5

### Where did you install the app from?

Google Play

### Device information

CCwGTV

### Android version

12

### Jellyfin server version

10.8.9

**Comment by branches78** [comment]
You can find more info here: https://github.com/jellyfin/jellyfin-androidtv/issues/1753

**Comment by branches78** [comment]
Yeah, Auto Bitrate is not working very well but 120MB/s does work correctly, you can find more info here: https://github.com/jellyfin/jellyfin-androidtv/issues/1753

**Comment by branches78** [comment]
This issue hasn't been fixed, not even on the latest betas.

**Comment by branches78** [comment]
Still waiting for a fix...

**Comment by fice-t** [comment]
Would it make sense to treat same-network connections differently, as in the [web client](https://github.com/jellyfin-archive/jellyfin-apiclient-javascript/pull/424)?

**Comment by nielsvanvelzen** [comment]
No, LAN detection is not reliable. Especially when the Jellyfin is containerized or a reverse proxy is used. At this moment we do not recommend using the "auto" bitrate and is is disabled by default.

**Comment by Iiviin** [comment]
Just encountered this on Amazon fire stick 4K.

**Comment by Ulrar** [comment]
Same here

**Comment by nielsvanvelzen** [comment]
This issue is fixed in #3372 for 0.17.
